Styles/header.css

```diff
@@ -18,9 +18,12 @@
 }
 
 .nav-header .button {
-  height: fit-content;
+  height: 1.5rem;
+  width: 4rem;
+  text-align: center;
   background-color: wheat;
```

Comment:
You can give a variable to the `wheat` colour and use its hex code instead of directly the name